src/Mount.cpp

```diff
@@ -245,8 +245,8 @@ void Mount::configureRAStepper(byte pin1, byte pin2, int maxSpeed, int maxAccele
   // Use another AccelStepper to run the RA motor as well. This instance tracks earths rotation.
   _stepperTRK = new AccelStepper(AccelStepper::DRIVER, pin1, pin2);
 
-  _stepperTRK->setMaxSpeed(10000);
-  _stepperTRK->setAcceleration(5000);
+  _stepperTRK->setMaxSpeed(2000);
```

Comment:
Is there a reference (or calculation) we can use here instead of a magic number?

Reply:
You mean the max speed of 2000? Thats the upper limit of what Accelstepper can achieve

Reply:
Yes, `2000` does not convey a lot of information when reading the code though. Can `2000` be derived from an existing constant or `#define`? Can `2000` be converted into more meaningful units? etc.

Reply:
It's steps/second. You could make a constant like TwoThousandStepsPerSec... or add comments... or know what setSpeed() expects 😁

Reply:
Ok, then would it be reasonable to convert from something like the earth's rotation rate (360.985deg/day) and back down to the local stepper frame?

Reply:
No. Its purely the MAXIMUM steps per second that we can run the steppers at. It has no bearing on the earth or anything except maybe the interrupt frequency….

Reply:
So what's the benefit of changing it from 10000step/s to 2000step/s then? If it's based on our interrupt frequency we should make a concrete link to that, so that they can be kept in sync.

Reply:
There is no benefit, since we never issue speeds that high anyway.

Reply:
We could name it MAX_PHYSICAL_SPEED